experimental/traffic-portal/src/app/api/testing/cdn.service.ts

```diff
@@ -175,4 +175,21 @@ export class CDNService {
 		this.cdns[idx] = cdn;
 		return cdn;
 	}
+
+	/**
+	 * Queues or dequeues updates on a CDN's servers.
+	 *
+	 * @param cdn The CDN to queue or dequeue updates on.
+	 * @param action The action to perform on the CDN, either "queue" or "dequeue".
+	 */
+	public async queueCDNUpdates(cdn: ResponseCDN | number, action: "queue" | "dequeue"): Promise<CDNQueueResponse> {
+		const cdnId = typeof cdn === "number"
+			? cdn
+			: cdn.id;
+
+		return {
+			action,
+			cdnId,
+		};
+	}
```

Comment:
I know you had a test for this method at one point, but it seems to be gone now?

Reply:
Ehh, #7497 added `queueCDNUpdates()` and `dequeueCDNUpdates()` to `CDNService`, so it might as well just switch to that. Removed `queueCDNUpdates` in c625c7ef4f.